lib/internal/Magento/Framework/Image/Adapter/ImageMagick.php

```diff
@@ -241,7 +253,7 @@ public function crop($top = 0, $left = 0, $right = 0, $bottom = 0)
      * @param bool $tile
      * @return void
      * @throws \LogicException
-     * @throws \Exception
+     * @throws \Magento\Framework\Exception\LocalizedException
```

Comment:
Use imported class name

Reply:
But why? Extracted documentation is not aware of the imports. 
I see those everywhere in Magento Code:
```
    /**
     * Remove product from Bundle product option
     *
     * @param string $sku
     * @param int $optionId
     * @param string $childSku
     * @throws \Magento\Framework\Exception\NoSuchEntityException
     * @throws \Magento\Framework\Exception\InputException
     * @return bool
     */
    public function removeChild($sku, $optionId, $childSku);
```

Reply:
Let me discuss that. Let's keep it like this for now :-)